### PR TITLE
Update prodsbx to connected to peered BlueButton

### PR DESCRIPTION
**Pull request by ronaldheft-usds**
**Why**

Connected to the newly peered BlueButton URL.

### CODE REVIEW

File: dpc-aggregation/src/main/resources/prod-sbx.application.conf [hunk 1]
```diff
@@ -7,6 +7,7 @@ dpc.aggregation {
     password = dpc-safe
   }
   bbclient.keyStore.location = "/bb.keystore"
+  bbclient.serverBaseUrl = "https://internal-dpcwelb01-2074070868.us-east-1.elb.amazonaws.com/v1/fhir/"
```
Comment (nickrobison-usds) [inline dpc-aggregation/src/main/resources/prod-sbx.application.conf]: This should just be `https://internal-dpcwelb01-2074070868.us-east-1.elb.amazonaws.com`

Comment (ronaldheft-usds) [reply dpc-aggregation/src/main/resources/prod-sbx.application.conf]: @nickrobison-usds serverBaseUrl in `application.conf` includes `/v1/fhir/`. Is that incorrect as well?

Comment (nickrobison-usds) [reply dpc-aggregation/src/main/resources/prod-sbx.application.conf]: Honestly, not entirely sure, I think you're right though. Let's try it with the /v1/fhir

**Comment by ronaldheft-usds** [comment]
@nickrobison-usds Will be doing deploys to test later today to continue the work on the Jenkins deployment script. I can run a deploy with the peered URL in test. Should this also be set in test? Right now only the peered URL is used in prod-sbx.

**Comment by nickrobison-usds** [comment]
@ronaldheft-usds Let's set this up in test and validate there. I checked the peering was working this morning.

**Comment by ronaldheft-usds** [comment]
We're still having connectively issue in `test` and `prod-sbx`. Will be investigating more tomorrow.

**Comment by RickHawesUSDS** [comment]
DPC is not on-boarded into BFD. You'll need to hit the new CCS environment when you do. The domain will be `dpc.bfdcloud.net`.

**Comment by ronaldheft-usds** [comment]
This PR is no longer valid. We reconfigured things to accept a parameter via Docker for the BFD backend URL.

**Pull request closed without merging**